database/factories/ModelFactory.php

```diff
@@ -1,11 +1,24 @@
 <?php
 
 use Faker\Generator;
+use Chompy\Models\RockTheVoteLog;
 use Chompy\Models\RockTheVoteReport;
 
+$factory->define(RockTheVoteLog::class, function (Generator $faker) {
+    return [
+        'finish_with_state' => 'No',
+        'import_file_id' => $this->faker->randomDigitNotNull,
+        'pre_registered' => 'No',
+        'started_registration' => '2020-02-22 19:16:32 -0500',
```

Comment:
Would it be better to use `Carbon` for this faked date? You can set it to do dates in past if needed, but seems better than maybe introducing a typo into this very specific date. 🤔